cve_bin_tool/checkers/mysql.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for mysql CLI
+References:
+https://www.mysql.com/
+https://www.cvedetails.com/vulnerability-list/vendor_id-93/product_id-21801/Oracle-Mysql.html
+"""
+from . import Checker
+
+
+class MysqlChecker(Checker):
+
+    CONTAINS_PATTERNS = [
+        r"For information about MySQL products and services, visit:",
+        r"Oracle and/or its affiliates. All rights reserved.",
+        r"MYSQL_USER",
```

Comment:
I feel like this string might also be a false positive risk, because other programs may have stuff that changes mysql user tables.  Maybe we could find something longer and less likely to occur in normal usage of mysql?